README.md

```diff
@@ -21,59 +21,58 @@ Once on a terminal window or command line, for simplicity, run commands from the
 
 ### Python
 
-Check for python installed. 
-This tool is [tested with Python 3.6, 3.7, and 3.8](https://app.travis-ci.com/dimagi/commcare-export).
+Check which Python version is installed.
 
+This tool is tested with Python versions from 3.8 to 3.12.
 
 ```shell
 $ python --version
 $ python3 --version
 ```
-If python is installed, all of its available versions would be listed.
+If Python is installed, its version will be listed.
 
-If python isn't installed, Install Python 3.8 from [this link](https://www.python.org/downloads/). 
+If Python isn't installed, [download and install](https://www.python.org/downloads/) the latest release. 
 
 ## Virtualenv (Optional)
 
 It is recommended to set up a virtual environment for CommCare Export
-to avoid conflicts with other python applications.
+to avoid conflicts with other Python applications.
 
 More about virtualenvs on https://docs.python.org/3/tutorial/venv.html
 
 Setup a virtual environment using:
 
 ```shell
-$ python3.8 -m venv .commcare-export-venv # update python version for the one installed
+$ python3 -m venv .venv
```

Comment:
Any reason to change that back to venv?
I had made that change earlier to make the name more specific and avoid conflicts with any other setup since that is the name suggested by virtualenv documentation by default. We use the same name for CommCareHQ, `venv`.

Reply:
Personally, I like "venv". (Some people seem to like to hide the directory.) I'll change it.

Reply:
I am happy with venv if you wish to keep that.

Reply:
84fe33b